### ISSUE TITLE
Automatically give ess-dive-admins access permissions to all drafts.

### ISSUE THREAD

**Issue by heshamkelbashandy** [heshamkelbashandy/ess-dive-catalog#55]
In GitLab by @vchendrix on Feb 1, 2018, 18:32

Automatically give ess-dive-admins access permissions to all drafts.  This should be configurable in the web application (most likely in `config.js`)

Adding the default permission will adding an access policy onto all of the created objects (metadata, resource map and data).  

## Example Access Policy
```xml
<accessPolicy>
<allow>
<subject>CN=ess-dive-admins,DC=dataone,DC=org</subject>
<permission>changePermission</permission>
</allow>
</accessPolicy>
```

Start examining `js/models/metadata/eml211/EML211.js` to see if there is a way to extend it to set a default accessPolicy value.

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Feb 1, 2018, 20:22

added ~3203215 label

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Feb 2, 2018, 15:23

assigned to @vchendrix

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Feb 5, 2018, 12:50

duplicate of #3

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Feb 8, 2018, 12:44

closed via commit 8a7f4e97e627624f1d5648fe8d68bf242dc4c45e

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Feb 8, 2018, 12:44

closed via commit f0ebe2130cf02a0f9228ef8ae7c059cea06852e3

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Feb 8, 2018, 12:44

closed via merge request !19
